### PR TITLE
Copy gitignore files.

### PR DESCRIPTION
**Pull request by iVilja**
#38

### CODE REVIEW

File: src/template.ts [hunk 1]
```diff
@@ -91,7 +91,7 @@ export async function copy(args: CopyConfig) {
     const targetPath = format(
       path.resolve(args.packageDir, relativePath),
       args.view
-    );
+    ).replace(/\/gitignore$/, '/.gitignore'); // https://github.com/uetchy/create-create-app/issues/38
```
Comment (uetchy) [inline src/template.ts:94]: For Windows, wrap targetPath in `slash` before replacing path

**Review by uetchy — CHANGES_REQUESTED** [review]

**Comment by uetchy** [comment]
Also rebase to the latest commit to ensure GitHub Actions will run the tests for Windows

**Comment by iVilja** [comment]
Thanks. Done.

**Comment by uetchy** [comment]
Thanks! @all-contributors add @iVilja for code

**Pull request merged**